apps/condo/domains/common/hooks/useHotUpdate.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react'
+import { notification } from 'antd'
+import Router from 'next/router'
+import { useCurrentBuild } from './useCurrentBuild'
+import { usePrevious } from './usePrevious'
+import { useIntl } from 'react-intl'
+
+const DELAY_IN_SECONDS = 5
+
+
+export function useHotUpdate (): void {
```

Comment:
It looks really clean, but i think a quick explanation of whats going on here and how it works would help

Reply:
added